### DISCUSSION TITLE
Update blocking VMware?

### DISCUSSION THREAD

**Discussion by Delzonfire** [complexorganizations/wireguard-manager#310] [General]
Not sure if intended, but build is blocking running the script in ESXi.

**Comment by Rebelllious** [comment]
The latest main branch code supports VMware virtualization. Can you post the output that you get when it's failing on ESXi?

**Comment by Delzonfire** [comment]
Running the following command ->  bash /usr/local/bin/wireguard-manager.sh , and get the output below

"vmware virtualization is not supported (yet)."

**Comment by Rebelllious** [comment]
_grep -i vmware wireguard-manager.sh_

What is the output for the above? Make sure to navigate to the directory where the script is stored or include the full path to it.

**Comment by Delzonfire** [comment]
Had to get the environment up quickly again for use, but I've made a snap of the VM and will test and report back here. 

No changes were made when the issue occurred, believe I did have automatic updates on.

**Comment by Rebelllious** [comment]
https://github.com/complexorganizations/wireguard-manager/blob/main/wireguard-manager.sh#L72

This has support for VMware. Are you sure you grabbed the script from the main branch and not from some work-in-progress one?

**Reply by Delzonfire** [reply]
I'm wondering if it grabbed the WIP version, but I had not actually done any updates/adjustments. Just logged in and that occurred.
